### ISSUE TITLE
Store and return testDate on verify code API

### ISSUE THREAD

**Issue by mikehelmick** [google/exposure-notifications-verification-server#882]
### TL;DR

Mobile apps would like to differentiate between testDate and symptomDate for purposes of prompting the user.

Currently this is collapsed to a single date that is stored and transported through the verify response, verification token, and verification certificate.

### Design

**Proposal**

1) add `TestDate` to the verification code struct
2) store this in the DB
3) when the verificaiton token is minted, expand the subject to contain `diagnosis.symptomdate.testdate`
4) verification token structure will remain the same - a single date called symptom date, that can be either.

/kind enhancement
